### PR TITLE
[13.0][MIG] report_xlsx_helper

### PR DESCRIPTION
**Pull request by kongrattapong**
Standard migration.

### CODE REVIEW

**Review by ps-tubtim — APPROVED** [review]

File: report_xlsx_helper/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+This module requires report_xlsx version 11.0.1.0.3 or higher.
```
Comment (kittiu) [inline report_xlsx_helper/readme/INSTALL.rst]: Is this state ment still valid?
I think it would need version 13.0.xxx isn't it?

Comment (kongrattapong) [reply report_xlsx_helper/readme/INSTALL.rst]: Thank for your review, Fixed.

**Review by kittiu — APPROVED** [review]

**Comment by kittiu** [comment]
cc @luc-demeyer would you review this :)

**Comment by Saran440** [comment]
@kongrattapong please cherry-pick this PR https://github.com/OCA/reporting-engine/pull/364

**Review by luc-demeyer — APPROVED** [review]

**Comment by luc-demeyer** [comment]
code review & testing ok.
I think we can go ahead and merge this PR.

**Review by pankk — APPROVED** [review]
Code review: virtually no changes to the module other than formatting.

**Comment by kittiu** [comment]
@kongrattapong would you change title from xlsx_report_helper -> report_xlsx_helper
Otherwise, other people may repeat this work if he/she can't find this.

**Comment by luc-demeyer** [comment]
can someone of the committers merge this one please ?

**Review by christdum — APPROVED** [review]

**Review by Saran440 — APPROVED** [review]

**Comment by luc-demeyer** [comment]
@lmignon can you merge ?

**Comment by luc-demeyer** [comment]
@lmignon Thanks!

**Pull request merged**